client/public/styles/style.css

```diff
@@ -1,6 +1,7 @@
 body {
     min-width: 1020px;
     word-wrap: break-word;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
:⭕️ why did you choose this font stack? Putting specific fonts as fallbacks after `sans-serif` doesn't make lots of sense, imo